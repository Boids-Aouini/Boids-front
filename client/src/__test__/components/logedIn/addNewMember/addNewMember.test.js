import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';
import AddNewMember from '../../../../components/logedIn/addNewMember/addNewMember';

let mockStore = configureStore();
let initState = {};
let store = mockStore(initState);

describe('<AddNewMember />', () => {
    describe('render()', () => {
        test('should render without crashing', () => {
            shallow(<AddNewMember store={store} />)
        })
        test('render component', () => {
            let wrapper = shallow(<AddNewMember store={store} />)
            let component = wrapper.dive();
            expect(toJson(component)).toMatchSnapshot()
        })

    })

})
