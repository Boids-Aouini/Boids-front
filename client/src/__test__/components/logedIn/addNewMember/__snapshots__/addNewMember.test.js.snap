// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddNewMember /> render() render component 1`] = `
<AddNewMember
  addMember={[Function]}
  addedMember={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
